### PR TITLE
Update cloud storage docs

### PR DESCRIPTION
**Pull request by gordonhart**
- Add icons
- Update index page
- Fix JSON typo

### CODE REVIEW

**Review by ly-kolena — APPROVED** [review]

File: docs/advanced-usage/connecting-cloud-storage/google-cloud-storage.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+---
+icon: simple/googlecloud
```
Comment (dylangrandmont) [inline docs/advanced-usage/connecting-cloud-storage/google-cloud-storage.md:2]: Where does this icon come from? (/ where can I find it?)

Comment (dylangrandmont) [reply docs/advanced-usage/connecting-cloud-storage/google-cloud-storage.md:2]: Interesting. I guess several icon sets come bundled here. Good to know https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/?h=icons

File: docs/advanced-usage/connecting-cloud-storage/s3-compatible.md [hunk 2]
```diff
@@ -1,11 +1,14 @@
 ---
+icon: simple/amazons3
```
Comment (dylangrandmont) [inline docs/advanced-usage/connecting-cloud-storage/s3-compatible.md]: There is a minIO icon. We may want to use that here instead so this icon remains reserved for the s3 integration type

Comment (gordonhart) [reply docs/advanced-usage/connecting-cloud-storage/s3-compatible.md]: Updated 👍 

Also added Hitachi as a compatible service — does this look good to you?

**Review by dylangrandmont — APPROVED** [review]
🚢

**Pull request merged**